PROJECT

```diff
@@ -59,6 +59,7 @@ resources:
   group: configuration
   kind: IngressClassParameters
   path: github.com/kong/kubernetes-ingress-controller/pkg/apis/configuration/v1alpha1
+  plural: IngressClassParameterses
```

Comment:
It was never used on multiple levels:
- https://github.com/Kong/kubernetes-ingress-controller/commit/193e36c8fdd19107df74a7fb4dfaff128a74a199 is only in main so far. We have never released this.
- https://github.com/Kong/kubernetes-ingress-controller/pull/2535 did not include the manifest in the CRD kustomization. You would have only installed it if you applied it directly from `config/crd/bases`.
- If you had applied the CRD and enabled the controller, the issue that this name change fixes (among others) prevented the controller from interacting with the CRs, and eventually made it crash.

Reply:
Yeah.. So this is sort of a consequence of releasing "not often enough" to get "new features" to the hands of our users.

As I was reading this comment I thought about adding a CRD support matrix table in docs.konghq.com so that our users would be able to see what CRDs in which versions are supported in particular KIC versions.

Basically, slapping one more table in here: https://docs.konghq.com/kubernetes-ingress-controller/latest/references/version-compatibility/#kubernetes